aptos-move/aptos-gas/src/transaction.rs

```diff
@@ -83,7 +83,7 @@ define_gas_parameters_for_transaction!(
     [
         max_transaction_size_in_bytes,
         "max_transaction_size_in_bytes",
-        8192
+        10 * 1024 * 1024
```

Comment:
should it be 12 or 10?

Reply:
This should be 10. 12 is the max request length